### ISSUE TITLE
fine-tune code

### ISSUE THREAD

**Issue by JunMa11** [adithya-s-k/YoloGemma#1]
Dear @adithya-s-k ,

Thanks for the great repo.

Do you have any plans to share a fine-tuning script on customized datasets?

**Comment by adithya-s-k** [comment]
Hey the project is built around Paligemma

Here are some refrences
https://ai.google.dev/gemma/docs/paligemma
https://huggingface.co/blog/paligemma

Finetuning guide:
https://github.com/NielsRogge/Transformers-Tutorials/blob/master/PaliGemma/Fine_tune_PaliGemma_for_image_%3EJSON.ipynb

Hope this helps

**Comment by JunMa11** [comment]
Hi @adithya-s-k ,

Thanks for your swift response!

I'll prepare my data as follows:
- object detection dataset: https://huggingface.co/docs/datasets/en/image_dataset#object-detection
- segmentation dataset: https://huggingface.co/docs/datasets/en/semantic_segmentation

Would you happen to have any suggested script to convert the bounding box and segmentation mask to tokens (that can be used in Paligemma)?
